### DISCUSSION TITLE
Speed in Evaluation metrics

### DISCUSSION THREAD

**Discussion by sudarshan-koirala** [explosion/spaCy#8823] [Help: Coding & Implementations] [answered]
What does `SPEED` mean in the evaluation metrics ? Tried to find but didn't see documents related to it.

![image](https://user-images.githubusercontent.com/14214659/127303024-91e7c603-f3c4-4a0b-809f-3f2d11bf15a9.png)

**Comment by polm** [comment, marked as answer]
I think that's words per second and is calculated [here](https://github.com/explosion/spaCy/blob/8547514aa4d17bd5f457d2b58d6c3efc3748520d/spacy/language.py#L1391). We should definitely make that more prominent in the documentation, thanks for pointing it out!
